### ISSUE TITLE
write() argument must be str, not numpy.ndarray

### ISSUE THREAD

**Issue by 17702513221** [yywbxgl/rtsp_opencv_demo#2]

**Comment by sean-wade** [comment]
same question, did you solve it? Thanks

**Comment by ghost** [comment]
> same question, did you solve it? Thanks

Yes I did, If you're writing to file, The code `str(<numpy.ndarray value>)` solves the issue. if this solves it for you, consider closing this issue.
